config/defaults.json

```diff
@@ -3,7 +3,8 @@
     "port": 8200,
     "hostname": "localhost",
     "renewable": true,
-    "ttl": "5m"
+    "ttl": 300,
+    "explicit_max_ttl": 7200
```

Comment:
Is having a default value here going to be problematic for upgrades when we add support for Vault 0.7 and [periodic tokens](https://www.vaultproject.io/docs/concepts/tokens.html)?